build-tools/create-packs/Directory.Build.targets

```diff
@@ -254,15 +254,19 @@
 
   <Target Name="InstallManifestAndDependencies">
     <PropertyGroup>
+      <_EmptyDepsWorkloadName>android.deps.workload</_EmptyDepsWorkloadName>
       <_LocalSdkManifestsFolder>$(BuildOutputDirectory)lib\sdk-manifests\$(DotNetSdkManifestsFolder)\</_LocalSdkManifestsFolder>
       <_LocalAndroidManifestFolder>$(_LocalSdkManifestsFolder)microsoft.net.sdk.android\</_LocalAndroidManifestFolder>
-      <_EmptyWorkloadDir>$(_LocalSdkManifestsFolder)android.deps.workload\</_EmptyWorkloadDir>
+      <_EmptyWorkloadDir>$(_LocalSdkManifestsFolder)$(_EmptyDepsWorkloadName)\</_EmptyWorkloadDir>
       <_EmptyWorkloadJsonContent>
 <![CDATA[
 {"version": "0.0.1", "workloads": { "android-deps": { "extends" : [ "microsoft-net-runtime-android", "microsoft-net-runtime-android-aot" ] } } }
 ]]>
       </_EmptyWorkloadJsonContent>
     </PropertyGroup>
+    <ItemGroup>
+      <_WorkloadManifestsTxt Include="$(DotNetPreviewPath)sdk\*\IncludedWorkloadManifests.txt" />
```

Comment:
Looking at the code we might also be able to delete this file?

https://github.com/dotnet/sdk/blob/09b31215867d1ffe4955fd5b7cd91eb552d3632c/src/Resolvers/Microsoft.NET.Sdk.WorkloadManifestReader/SdkDirectoryWorkloadManifestProvider.cs#L44-L48

Reply:
I tried testing this after removing the `IncludedWorkloadManifests.txt` file and `workload search` and `workload install` failed to recognize any workload IDs. I think a better fix has been added #7203.